### PR TITLE
AzureDatabricks-workspace-with-managed-disk-customer-managed-keys

### PR DESCRIPTION
**Pull request by v-gsapaharam**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by v-gsapaharam** [comment]
Greetings Team,  could you please investigate bug# https://github.com/Azure/arm-ttk/issues/713.

**Comment by v-gsapaharam** [comment]
Hello Brian, could you please investigate the bug# https://github.com/Azure/arm-ttk/issues/713

File: quickstarts/microsoft.databricks/databricks-workspace-with-managed-disk-customer-managed-keys/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+               "name": "[parameters('pricingTier')]"
+           },
+           "properties": {
+               "managedResourceGroupId": "[concat(subscription().id, '/resourceGroups/', variables('managedResourceGroupName'))]",
+               "encryption": {
+                   "entities": {
+                       "managedDisk": {
+                           "keySource": "Microsoft.Keyvault",
+                           "keyVaultProperties": {
+                               "keyVaultUri": "[concat('https://', parameters('keyVaultName'), environment().suffixes.keyvaultDns)]",
```
Comment (bmoore-msft) [inline quickstarts/microsoft.databricks/databricks-workspace-with-managed-disk-customer-managed-keys/azuredeploy.json:84]:
```suggestion
                               "keyVaultUri": "[uri(concat('https://', parameters('keyVaultName'), environment().suffixes.keyvaultDns), '/')]",
```

File: quickstarts/microsoft.databricks/databricks-workspace-with-managed-disk-customer-managed-keys/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+       {
+           "type": "Microsoft.Databricks/workspaces",
+           "apiVersion": "2022-04-01-preview",
+           "name": "[parameters('workspaceName')]",
+            "location": "[parameters('location')]",
+           "sku": {
+               "name": "[parameters('pricingTier')]"
+           },
+           "properties": {
+               "managedResourceGroupId": "[concat(subscription().id, '/resourceGroups/', variables('managedResourceGroupName'))]",
```
Comment (bmoore-msft) [inline quickstarts/microsoft.databricks/databricks-workspace-with-managed-disk-customer-managed-keys/azuredeploy.json:78]:
```suggestion
               "managedResourceGroupId": "[subscriptionResourceId('Microsoft.Resources/resourceGroups', variables('managedResourceGroupName'))]",
```

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Pull request merged**